modules/Netbox.py

```diff
@@ -0,0 +1,64 @@
+import pynetbox
+
+
+class Netbox:
+    def __init__(self, nb_config):
+        self.nb_config = nb_config
+        self.nb = self._login()
+
+    def _login(self):
+        """
+        Login to netbox
+        :return:
+        """
+        nb = pynetbox.api(url=self.nb_config["netbox_url"], token=self.nb_config["netbox_token"])
```

Comment:
Try except this too please, logins can always go wrong. Do proper error handling in case this fires back.

Reply:
Here the object gets created any way if the token is wrong. So added try catch when fetching the information. please check. Thanks

Reply:
ok just had a look at the api, it is just the handler object. The method name then should indicate this. Please name it `_create_handle` or similar.

Reply:
Done